osu.Game/Localisation/GraphicsSettingsStrings.cs

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class GraphicsSettingsStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.GraphicsSettings";
+
+        /// <summary>
+        /// "Graphics"
+        /// </summary>
+        public static LocalisableString GraphicsSectionHeader => new TranslatableString(getKey(@"graphics_section_header"), @"Graphics");
+
+        /// <summary>
+        /// "Renderer"
+        /// </summary>
+        public static LocalisableString RendererHeader => new TranslatableString(getKey(@"renderer_header"), @"Renderer");
+
+        /// <summary>
+        /// "Frame limiter"
+        /// </summary>
+        public static LocalisableString FrameLimiter => new TranslatableString(getKey(@"frame_limiter"), @"Frame limiter");
+
+        /// <summary>
+        /// "Threading mode"
+        /// </summary>
+        public static LocalisableString ThreadingMode => new TranslatableString(getKey(@"threading_mode"), @"Threading mode");
+
+        /// <summary>
+        /// "Show FPS"
+        /// </summary>
+        public static LocalisableString ShowFPS => new TranslatableString(getKey(@"show_fps"), @"Show FPS");
```

Comment:
Haven't done a full pass on this (will take some time later) but you need to add `FPS` as an acronym in the dotsettings file.